services/QuillLMS/app/services/skills_report/question_responses_for_classroom.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module SkillsReport
+  class QuestionResponsesForClassroom < ApplicationService
```

Comment:
This and the related `QuestionResponseForClassroomBySkill` service need a decent amount of refactoring, I believe.  Just for clarity around options and to avoid all the string interpolation in the SQL.  I welcome suggestions on improvements, but would like to get these merged before pursuing them.